### DISCUSSION TITLE
Dependency data is not used when tracking Azure Service Bus requests

### DISCUSSION THREAD

**Discussion by stijnmoreels** [arcus-azure/arcus.observability#266] [General]
We currently don't use the dependency data from the `DependencyMeasurement` instance when tracking Azure Service Bus requests. Is this something that we should consider adding? Like by-default in the telemetry context? Or should we just ignore this.

Relates to #250

**Comment by fgheysels** [comment]
We have decided to create a `DurationMeasurement` type which will be used for measuring dependencies and requests.
See this issue: https://github.com/arcus-azure/arcus.observability/issues/283

This has been implemented in Arcus.Observability 2.4.0
